android/app/src/main/kotlin/com/zulip/flutter/ZulipPlugin.kt

```diff
@@ -51,6 +55,13 @@ private class AndroidNotificationHost(val context: Context)
                 Bundle().apply { it.forEach { (k, v) -> putString(k, v) } } ) }
             smallIconResourceName?.let { setSmallIcon(context.resources.getIdentifier(
                 it, "drawable", context.packageName)) }
+            groupKey?.let { setGroup(it) }
```

Comment:
nit: match here the ordering of the parameters above (following up on https://github.com/zulip/zulip-flutter/pull/703#discussion_r1643507237 )